core/controllers/dashboard.py

```diff
@@ -257,6 +258,204 @@ def _round_average_ratings(rating):
         self.render_json(self.values)
 
 
+class LearnerDashboardPage(base.BaseHandler):
```

Comment:
I don't think that last convention is a good long-term solution, unfortunately. We used to have just one dashboard before, so "dashboard" was unambiguous; now we have two. So I think we would need to update the naming in any case.